source/_integrations/facebox.markdown

```diff
@@ -12,7 +12,7 @@ The `facebox` image processing platform allows you to detect and recognize faces
 
 ## Setup
 
-Facebox runs in a Docker container and it is recommended that you run this container on a machine with a minimum of 2 GB RAM. On your machine with Docker, run the Facebox container with:
+Facebox runs in a Docker container and it is recommended that you run this container on a x86 machine with a minimum of 2 GB RAM (an ARM version is not available). On your machine with Docker, run the Facebox container with:
```

Comment:
We should link to their specs, instead of maintaining their system requirements in our documentation.

Reply:
Yes, but there are no infos about that on their web page. 
https://machinebox.io/docs/facebox
https://docs.veritone.com/#/developer/machine-box/boxes/tagbox/recognizing-images

The only info I found after a long time was here: 
https://github.com/machinebox/issues/issues/15

I just don't want that anybody else needs to try as long as I did... :-)

Seb

Reply:
Alright, in that case, let's do this! 👍